plugins/auth-backend-module-guest-provider/package.json

```diff
@@ -41,7 +45,5 @@
     "@backstage/config": "workspace:^",
     "express": "^4.18.2"
   },
-  "files": [
-    "dist"
-  ]
+  "configSchema": "config.d.ts"
```

Comment:
so this and the `files` addition above are the only changes related to the fix :) the rest is just hackdays mood happening